### ISSUE TITLE
WS02 - Question 2 - Swapping cross entropy loss for MSE loss

### ISSUE THREAD

**Issue by Aseamann** [Graylab/DL4Proteins-notebooks#5]
Swapping to MSE loss is not directly intuitive. MSEloss in PyTorch takes labels that are one-hot encoded as default. To do this, I generated a separate variable "y_onehot". It may be helpful to clarify this in a "hint" box or something along with this question.

```
# one hot encode y
y_onehot = torch.nn.functional.one_hot(y, num_classes=3)
y_onehot = y_onehot.type(torch.float32)
```
